_format/1.1/index.md

```diff
@@ -79,6 +234,17 @@ than `ext` or `profile`. If every instance of that media type is modified by the
 `ext` parameter and each contains at least one unsupported extension URI, the
 server **MUST** also respond with a `406 Not Acceptable`.
 
+If the `ext` parameter contains at least one unsupported extension URI, the
+server **MUST** also respond with a `406 Not Acceptable`.
```

Comment:
This sentence appears to repeat the last sentence of the preceding paragraph.

Reply:
Agreed. Removed.